src/main/resources/hudson/tasks/junit/JUnitResultArchiver/config.jelly

```diff
@@ -46,4 +46,7 @@ THE SOFTWARE.
     <f:entry title="${%Allow empty results}" field="allowEmptyResults">
         <f:checkbox default="false" title="${%Do not fail the build on empty test results}"/>
     </f:entry>
+    <f:entry title="${%Make the build UNSTABLE}" field="makeUnstable">
```

Comment:
this needs help text:
https://github.com/nguoianphu/junit-plugin/tree/makeUnstable/src/main/resources/hudson/tasks/junit/JUnitResultArchiver

Reply:
Will fix it. Thanks